src/server/index.js

```diff
@@ -120,6 +120,23 @@ router.post('/boards', function(req, res) {
   }
 });
 
+router.delete('/boards/:id', function(req, res) {
+  let username = jwtVerify(req.headers.token);
+  let boardId = req.params.id;
+
+  if (!username) {
+    res.status(STATUS_FORBIDDEN).json({message: 'Please login first!'});
+  } else {
+    boards.deleteBoardById(username, boardId, function(result) {
+      if (result === 'error' || !result || result.result.n === 0) {
```

Comment:
i think if `result.result.n === 0` is true it's not an internal server error. it's still a valid request. it might say already deleted. but definitely not an internal server error